tests/build/jvm-build.go

```diff
@@ -34,7 +34,7 @@ var (
 	testProjectRevision = utils.GetEnv("JVM_BUILD_SERVICE_TEST_REPO_REVISION", "34da5a8f51fba6a8b7ec75a727d3c72ebb5e1274")
 )
 
-var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jvm-build", "HACBS"), func() {
+var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("build-service", "jvm-build"), func() {
```

Comment:
instead of `build-service` can we use seperate label like `jvm-build-service` here. these tests are separate for [repo](https://github.com/redhat-appstudio/jvm-build-service) and [jvm-build-service component](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/jvm-build-service) in infra-deployment

Reply:
I think Sushanta is correct here, this test is JBS specific, so it should have the `jvm-build-service` label.

I totally forgot that this and multi-platform test should have their own label. I will update it in the [jira ticket](https://issues.redhat.com/browse/KONFLUX-3150)